apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -1,22 +1,13 @@
 <div
   class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
   [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px" />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px" />
+  <ng-content></ng-content>
 
   <section>
     <app-list-item
       *ngFor="let item of list"
-      [name]="item.firstname"
-      [id]="item.id"
-      [type]="type">
-    </app-list-item>
+      [name]="getName(item)"
+      (delete)="delete(item)"></app-list-item>
```

Comment:
What if I want to add an icon only for CityCard ? How will you handle that case ? To make it more customizable, the parent should handle the display. 

Look at `ngTemplateOutlet`